docs/versioned-plugins/index.asciidoc

```diff
@@ -11,10 +11,10 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 [[logstash-plugin-reference]]
 = Versioned Plugin Reference
 
-include::inputs.asciidoc[]
+include::inputs-index.asciidoc[]
```

Comment:
The filenames used in the index file are not correct. We need to fix this in the index.asciidoc file.